src/app/components/audio-recordings/pages/download/download.component.ts

```diff
@@ -0,0 +1,389 @@
+          our servers, and you can think of it as being similar to your username
+          and password.
+          <strong class="text-danger">
+            Treat this code like your password, and do not ever share it with
+            anyone!
+          </strong>
+          You can find the auth token in your
+          <a [strongRoute]="profile.route">{{ profile.label }}</a> page, under
+          the "Authentication Token" section. Copy the auth token, and keep it
+          for the next step<br />
+          You should perform this step whenever you wish to run the script as
+          the auth token can expire if left for a long enough period of time.
+        </p>
+
+        <h3>4. Run the script</h3>
+
+        <p>
+          Finally we can run the script. Read through the documentation of the
+          script for more specific advice/use cases. To run the command, take
+          the terminal you setup in the previous steps, and run the following
+          command:
+          <code>./download_audio_files.ps1</code>
+        </p>
+      </section>
+    </ng-container>
+  `,
+})
+class DownloadAudioRecordingsComponent
+  extends withUnsubscribe(PageComponent)
+  implements OnInit
+{
+  public fields: FormlyFieldConfig[];
+  public model: Model = {};
+  public models: ResolvedModelList;
+  public href = "";
+  public failure: boolean;
+  public contactUs = contactUsMenuItem;
+  public profile = myAccountMenuItem;
+
+  public constructor(
+    private config: ConfigService,
+    private route: ActivatedRoute,
+    private recordingsApi: AudioRecordingsService
+  ) {
+    super();
+  }
+
+  public ngOnInit(): void {
+    const models = retrieveResolvers(new PageInfo(this.route.snapshot.data));
+    if (!hasResolvedSuccessfully(models)) {
+      this.failure = true;
+      return;
+    }
+    this.models = models;
+    this.updateHref();
+
+    const hideProjects = this.config.settings.hideProjects;
+    this.fields = [];
+
+    if (!hideProjects) {
+      if (this.project) {
+        this.fields.push(this.getProjectFormInput());
+      } else {
+        // TODO Future work, read project list and make typeahead
+      }
+    }
+
+    if (this.region) {
+      this.fields.push(this.getRegionFormInput(hideProjects));
+    } else {
+      // TODO Future work, read regions/sites list and make typeahead
+    }
+
+    if (this.site) {
+      this.fields.push(this.getSiteFormInput(hideProjects));
+    } else {
+      // TODO Future work, read sites list and make typeahead
+    }
+
+    this.fields.push(...this.dateSelectionRange());
+  }
+
+  public get project(): Project {
+    return this.models[projectKey] as Project;
+  }
+  public get region(): Region {
+    return this.models[regionKey] as Region;
+  }
+  public get site(): Site {
+    return this.models[siteKey] as Site;
+  }
+
+  private getProjectFormInput(): FormlyFieldConfig {
+    return {
+      key: "projectId",
+      type: "select",
```

Comment:
just makes these labels if we're not going to support changing them in this version